facebookshare.module

```diff
@@ -62,108 +31,24 @@ function facebookshare_block_view($delta = '') {
 
   switch ($delta) {
     case 'facebook_share':
-      $settings = variable_get('facebookshare_block_settings', array());
-      $block['content'] = facebookshare_content($settings);
+      $current_page = $base_url . DIRECTORY_SEPARATOR . current_path();
+      $share_link = 'https://www.facebook.com/sharer/sharer.php?u=' . $current_page;
+
+      $fb_link = l('Share on Facebook', $share_link, array(
+        'external' => TRUE,
+        'attributes' => array(
+          'class' => array(
+            'fb-share',
+          ),
+        )));
```

Comment:
I'm uncertain about the indentation here, better to run it through a `phpcs` tool or similar.

Reply:
P.S. - this is changed in NG's PR.